### PR TITLE
Added a getting started guide for custom monitoring metrics

### PR DESCRIPTION
**Pull request by sayanh**
Confirm these statements before you submit your pull request:

- [x] I have read and submitted the required [Contributor Licence Agreements](https://github.com/kyma-project/community/blob/master/CONTRIBUTING.md#agreements-and-licenses).
- [x] This pull request follows the contributing guidelines, recommended Git workflow, and templates.
- [x] I have tested my changes.
- [x] I have updated the relevant documentation.

### CODE REVIEW

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+Following ports are used in the pod:
+1. Port: **8080**, as envoy will capture traffic only for ports listed in containerPorts of the pod, `containerPort: 8080`, or the list of ports in the annotation `traffic.sidecar.istio.io/includeInboundPorts`. Thus, this port is a part of the service mesh and can be used for application needs.
+
+2. Port: **8081**, the excluded port from service mesh and is used for exposing metrics only. So the network traffic **bypasses** Envoy and goes straight to the container. In Kyma, use the suggested port `8081` to expose metrics.
+
+### Expose a sample metrics application
+
+In order to expose prometheus metrics in Golang the prometheus community provides a library, [client_golang](https://github.com/prometheus/client_golang).
+
+This example has taken a basic example where `Gauge` and `Counter` metrics are exported using the package `prometheus`.
```
Comment (klaudiagrz) [inline docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: ``This is a basic example where `Gauge` and `Counter` metrics are exported using the `prometheus` package.``

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+1. Deploy the sample metrics application
```
Comment (klaudiagrz) [inline docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: Dots are sad that you do not remember about them :( 
. . . . .

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    ```bash
+    kubectl get pods
+    NAME                             READY     STATUS    RESTARTS   AGE
+    sample-metrics-c9f998959-jd2fz   2/2       Running   0          2m
+    sample-metrics-c9f998959-kfbp8   2/2       Running   0          2m
+    sample-metrics-c9f998959-nnp2n   2/2       Running   0          2m
+    sample-metrics-c9f998959-vdnkn   2/2       Running   0          2m
+    ```
+
+2. Run the `port-forward` on the sample-metrics-8081 Service for port 8081 to check the metrics
```
Comment (klaudiagrz) [inline docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: ``Run the `port-forward` command on the `sample-metrics-8081` service for the`8081` port to check the metrics.``

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+    ```bash
+    kubectl port-forward svc/sample-metrics-8081 8081:8081
+    ```
+    Then open a browser and access [`http://localhost:8081/metrics`](http://localhost:8081/metrics)
```
Comment (klaudiagrz) [inline docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: Can we add more space between this sentence and the screenshot?

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+    ![metrics on port 8081](assets/sample-metrics-2.png)
+
+The source code for the sample application is [here](https://github.com/kyma-project/examples/blob/master/monitoring-custom-metrics/main.go).
```
Comment (klaudiagrz) [inline docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: ``Find the source code for the sample application [here](https://github.com/kyma-project/examples/blob/master/monitoring-custom-metrics/main.go). See the [package prometheus](https://godoc.org/github.com/prometheus/client_golang/prometheus) for the reference documentation. Read  
[this](https://prometheus.io/docs/concepts/metric_types/) documentation to learn more about the Prometheus metric types.``

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+### Access the exposed metrics in Prometheus
+
+ Run the `port-forward` command on the `core-prometheus` service:
+    
+```bash
+kubectl port-forward svc/core-prometheus -n kyma-system 9090:9090
+Forwarding from 127.0.0.1:9090 -> 9090
+Forwarding from [::1]:9090 -> 9090
+```
+All the **sample-metrics** endpoints appear as the [`Targets`](http://localhost:9090/targets#job-sample-metrics-8081) list.
```
Comment (klaudiagrz) [inline docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: Can we add more space between this sentence and the screenshot?

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+![Prometheus Dashboard](assets/pm-dashboard-1.png)
+
+Use either `cpu_temperature_celsius` or `hd_errors_total` in the **expression** field in [here](http://localhost:9090/graph) and click the `Execute` button to check the values scrapped by Prometheus.
```
Comment (klaudiagrz) [inline docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: ``Use either the `cpu_temperature_celsius` or `hd_errors_total` in the [**expression**](http://localhost:9090/graph) field. Click the **Execute** button to check the values scrapped by Prometheus.``

Also: some space between the text and the photo if possible

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+![Prometheus Dashboard](assets/pm-dashboard-2.png)
+
+### ServiceMonitor
+
+With the help of ServiceMonitor, Prometheus is able to reach the service. A ServiceMonitor is a specific CRD used by the prometheus operator to monitor services.
```
Comment (klaudiagrz) [inline docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: ``Prometheus can reach the service using ServiceMonitor. ServiceMonitor is a specific CRD used by the Prometheus operator to monitor services.``

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+
+In Kyma the prometheus server discovers all ServiceMonitors through a ```serviceMonitorSelector:``` matching the label ```prometheus: core```
```
Comment (klaudiagrz) [inline docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: ``In Kyma, the Prometheus server discovers all ServiceMonitors through the **serviceMonitorSelector** matching the `prometheus: core` label.``

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 10]
```diff
@@ -0,0 +1,149 @@
...
+
+```yaml
+  serviceMonitorSelector:
+    matchLabels:
+      prometheus: {{ .Values.prometheusLabelValue | default .Release.Name | quote }}
+{{- end }}
+```
+
+In our example, the ServiceMonitor selects ```selector:```with all Services matching a label, ```k8s-app: metrics```. The complete yaml is [here](https://github.com/kyma-project/examples/blob/master/monitoring-custom-metrics/deployment/service-monitor.yaml).
```
Comment (klaudiagrz) [inline docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: ``In this example, the ServiceMonitor selects a **selector** with all services matching the `k8s-app: metrics` label. Find the complete yaml [here](https://github.com/kyma-project/examples/blob/master/monitoring-custom-metrics/deployment/service-monitor.yaml).``

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+
+In Kyma we have a template, [kyma/resources/core/charts/monitoring/charts/prometheus/templates/servicemonitors.yaml](https://github.com/kyma-project/kyma/blob/master/resources/core/charts/monitoring/charts/prometheus/templates/servicemonitors.yaml), which is ```kind: List``` to discover a list of ServiceMonitors.
```
Comment (klaudiagrz) [inline docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: ``In Kyma, there is a [template](https://github.com/kyma-project/kyma/blob/master/resources/core/charts/monitoring/charts/prometheus/templates/servicemonitors.yaml), which serves to discover a list of ServiceMonitors.`` -> I hope it's technically correct

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 12]
```diff
@@ -0,0 +1,149 @@
...
+
+### Adding Custom Dashboard in Grafana
```
Comment (klaudiagrz) [inline docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: ``### Add a Custom Dashboard in Grafana``

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 13]
```diff
@@ -0,0 +1,149 @@
...
+
+As mentioned, Kyma includes a set of dashboards. However, users can create their own **Grafana Dashboard** by using the Grafana UI as the dashboards are persisted even after the pod restarts.
```
Comment (klaudiagrz) [inline docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: ``Out of the box, Kyma includes a set of dashboards. The users can create their own **Grafana Dashboard** by using the Grafana UI. The dashboards persist even after the Pod restarts.``

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 14]
```diff
@@ -0,0 +1,149 @@
...
+
+For details of how to create dashboards in Grafana, see the following:
```
Comment (klaudiagrz) [inline docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: `For details on how to create dashboards in Grafana, see the following documents:`

Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md (hunk 13 above)
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md (hunk 12 above)
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md (hunk 11 above)
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: yes it is, changed

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md (hunk 10 above)
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md (hunk 9 above)
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md (hunk 8 above)
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md (hunk 7 above)
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md (hunk 6 above)
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: added a line break

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md (hunk 5 above)
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md (hunk 4 above)
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: added a line break

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md (hunk 3 above)
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md (hunk 2 above)
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md (hunk 1 above)
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 15]
```diff
@@ -0,0 +1,149 @@
...
+More details on installing the Istio Sidecar is here [here](https://istio.io/docs/setup/kubernetes/sidecar-injection/#policy).
+
+Following ports are used in the pod:
+1. Port: **8080**, as envoy will capture traffic only for ports listed in containerPorts of the pod, `containerPort: 8080`, or the list of ports in the annotation `traffic.sidecar.istio.io/includeInboundPorts`. Thus, this port is a part of the service mesh and can be used for application needs.
+
+2. Port: **8081**, the excluded port from service mesh and is used for exposing metrics only. So the network traffic **bypasses** Envoy and goes straight to the container. In Kyma, use the suggested port `8081` to expose metrics.
+
+### Expose a sample metrics application
+
+In order to expose prometheus metrics in Golang the prometheus community provides a library, [client_golang](https://github.com/prometheus/client_golang).
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 16]
```diff
@@ -0,0 +1,149 @@
...
+      annotations:
+        sidecar.istio.io/inject: "true"
+```
+
+More details on installing the Istio Sidecar is here [here](https://istio.io/docs/setup/kubernetes/sidecar-injection/#policy).
+
+Following ports are used in the pod:
+1. Port: **8080**, as envoy will capture traffic only for ports listed in containerPorts of the pod, `containerPort: 8080`, or the list of ports in the annotation `traffic.sidecar.istio.io/includeInboundPorts`. Thus, this port is a part of the service mesh and can be used for application needs.
+
+2. Port: **8081**, the excluded port from service mesh and is used for exposing metrics only. So the network traffic **bypasses** Envoy and goes straight to the container. In Kyma, use the suggested port `8081` to expose metrics.
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 17]
```diff
@@ -0,0 +1,149 @@
...
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true"
+```
+
+More details on installing the Istio Sidecar is here [here](https://istio.io/docs/setup/kubernetes/sidecar-injection/#policy).
+
+Following ports are used in the pod:
+1. Port: **8080**, as envoy will capture traffic only for ports listed in containerPorts of the pod, `containerPort: 8080`, or the list of ports in the annotation `traffic.sidecar.istio.io/includeInboundPorts`. Thus, this port is a part of the service mesh and can be used for application needs.
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: Absolutely, changed

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 18]
```diff
@@ -0,0 +1,149 @@
...
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true"
+```
+
+More details on installing the Istio Sidecar is here [here](https://istio.io/docs/setup/kubernetes/sidecar-injection/#policy).
+
+Following ports are used in the pod:
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 19]
```diff
@@ -0,0 +1,149 @@
...
+
+```yaml
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true"
+```
+
+More details on installing the Istio Sidecar is here [here](https://istio.io/docs/setup/kubernetes/sidecar-injection/#policy).
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: `sidecar` it is

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 20]
```diff
@@ -0,0 +1,149 @@
...
+To enable sidecar injection for all the pods running in the `default` Namespace, run the command below.
+
+```bash
+kubectl label namespace default istio-injection=enabled
+namespace "default" labeled
+```
+
+More details on deploying your application with Istio is [here](https://istio.io/docs/setup/kubernetes/quick-start#deploy-your-application) .
+
+It is needed to add the `sidecar.istio.io/inject` annotation with value `true` to the pod template spec to enable injection as done [here](https://github.com/kyma-project/examples/blob/master/monitoring-custom-metrics/deployment/deployment.yaml#L12).
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 21]
```diff
@@ -0,0 +1,149 @@
...
+for the `default` Namespace.
+
+To enable sidecar injection for all the pods running in the `default` Namespace, run the command below.
+
+```bash
+kubectl label namespace default istio-injection=enabled
+namespace "default" labeled
+```
+
+More details on deploying your application with Istio is [here](https://istio.io/docs/setup/kubernetes/quick-start#deploy-your-application) .
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 22]
```diff
@@ -0,0 +1,149 @@
...
+- Kyma as the target deployment environment.
+- Istio 0.8
+  - Sidecar injection must be enabled ```istio-injection: "enabled"```
+  - Mutual TLS enabled.
+
+## Installation
+
+### Configure Istio
+
+Sidecar injection must be enabled, `istio-injection: "enabled"`
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 23]
```diff
@@ -0,0 +1,149 @@
...
+
+1. Configure Istio
+2. Expose a sample application serving metrics on Port 8081.
+3. Access the exposed metrics in Prometheus
+
+## Prerequisites
+
+- Kyma as the target deployment environment.
+- Istio 0.8
+  - Sidecar injection must be enabled ```istio-injection: "enabled"```
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 24]
```diff
@@ -0,0 +1,149 @@
...
+  - Mutual TLS enabled.
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 25]
```diff
@@ -0,0 +1,149 @@
...
+
+This Getting started guide shows how to expose custom metrics to Prometheus with a Golang service in Kyma. To do so, follow these steps:
+
+1. Configure Istio
+2. Expose a sample application serving metrics on Port 8081.
+3. Access the exposed metrics in Prometheus
+
+## Prerequisites
+
+- Kyma as the target deployment environment.
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: 2. done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 26]
```diff
@@ -0,0 +1,149 @@
...
+type: Getting Started
+---
+
+## Overview
+
+This Getting started guide shows how to expose custom metrics to Prometheus with a Golang service in Kyma. To do so, follow these steps:
+
+1. Configure Istio
+2. Expose a sample application serving metrics on Port 8081.
+3. Access the exposed metrics in Prometheus
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 27]
```diff
@@ -0,0 +1,149 @@
+---
+title: Expose Custom Metrics in Kyma
+type: Getting Started
+---
+
+## Overview
+
+This Getting started guide shows how to expose custom metrics to Prometheus with a Golang service in Kyma. To do so, follow these steps:
+
+1. Configure Istio
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 28]
```diff
@@ -0,0 +1,149 @@
...
+2. Expose a sample application serving metrics on Port 8081.
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 29]
```diff
@@ -0,0 +1,149 @@
+---
+title: Expose Custom Metrics in Kyma
+type: Getting Started
+---
+
+## Overview
+
+This Getting started guide shows how to expose custom metrics to Prometheus with a Golang service in Kyma. To do so, follow these steps:
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/021-gs-monitoring-custom-metrics.md [hunk 30]
```diff
@@ -0,0 +1,149 @@
+---
```
[the remark this answers is not in the document]
Comment (sayanh) [reply docs/monitoring/docs/021-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/030-gs-monitoring-custom-metrics.md [hunk 31]
```diff
@@ -0,0 +1,145 @@
...
+1. Configure Istio.
+2. Expose a sample application serving metrics on `8081` port.
+3. Access the exposed metrics in Prometheus.
+
+## Prerequisites
+
+- Kyma as the target deployment environment
+- Istio 0.8
+  - sidecar injection enabled
+  - mutual TLS enabled.
```
Comment (klaudiagrz) [inline docs/monitoring/docs/030-gs-monitoring-custom-metrics.md]: no dot this time :D

File: docs/monitoring/docs/030-gs-monitoring-custom-metrics.md [hunk 32]
```diff
@@ -0,0 +1,145 @@
...
+    sample-metrics-c9f998959-kfbp8   2/2       Running   0          2m
+    sample-metrics-c9f998959-nnp2n   2/2       Running   0          2m
+    sample-metrics-c9f998959-vdnkn   2/2       Running   0          2m
+    ```
+
+2. Run the `port-forward` command on the `sample-metrics-8081` service for the`8081` port to check the metrics.
+    ```bash
+    kubectl port-forward svc/sample-metrics-8081 8081:8081
+    ```
+    Then open a browser and access [`http://localhost:8081/metrics`](http://localhost:8081/metrics)
```
Comment (klaudiagrz) [inline docs/monitoring/docs/030-gs-monitoring-custom-metrics.md]: ~Then~ `Open` ...

File: docs/monitoring/docs/030-gs-monitoring-custom-metrics.md [hunk 33]
```diff
@@ -0,0 +1,145 @@
...
+```yaml
+  serviceMonitorSelector:
+    matchLabels:
+      prometheus: {{ .Values.prometheusLabelValue | default .Release.Name | quote }}
+{{- end }}
+```
+
+In this example, the ServiceMonitor selects a **selector** with all services matching the `k8s-app: metrics` label. Find the complete yaml [here](https://github.com/kyma-project/examples/blob/master/monitoring-custom-metrics/deployment/service-monitor.yaml).
+
+In Kyma, there is a [template](https://github.com/kyma-project/kyma/blob/master/resources/core/charts/monitoring/charts/prometheus/templates/servicemonitors.yaml), which serves to discover a list of ServiceMonitors.
```
Comment (klaudiagrz) [inline docs/monitoring/docs/030-gs-monitoring-custom-metrics.md]: No comma after the link (sorry, my mistake)

File: docs/monitoring/docs/030-gs-monitoring-custom-metrics.md (hunk 31 above)
Comment (sayanh) [reply docs/monitoring/docs/030-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/030-gs-monitoring-custom-metrics.md (hunk 32 above)
Comment (sayanh) [reply docs/monitoring/docs/030-gs-monitoring-custom-metrics.md]: done

File: docs/monitoring/docs/030-gs-monitoring-custom-metrics.md (hunk 33 above)
Comment (sayanh) [reply docs/monitoring/docs/030-gs-monitoring-custom-metrics.md]: done

**Pull request merged**